dags/openshift_nightlies/manifest.yaml

```diff
@@ -146,6 +146,11 @@ platforms:
         config:
           install: rosa/ovn-osd.json
           benchmarks: data-plane.json
+      - name: ocm-api-load
+        schedule: "30 1 * * 1,3,5" # an hour gap for OSD to avoid OsdCcsAdmin key limit
```

Comment:
@venkataanil This job is scheduled to run along with `osd-ovn-data-plane` DAG, if they get triggered at same time high chance they would get in to access-key limit. OSD jobs are paused in sailplane airflow so shouldn't be a problem but better to change it.

Reply:
Strike that, in stage/integration the deployment is done more frequently with every PR. Production deployments happen weekly.

Reply:
I set schedule: "None" in latest commit